inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -140,14 +140,15 @@ void InLaneCruisingPlugin::plan_trajectory_callback(
       }
       else
       {
-        throw std::invalid_argument("Unable to Call Yield Plugin");
+        RCLCPP_WARN_STREAM(nh_->get_logger(), "UNABLE TO CALL YIELD PLUGIN!!!!");
```

Comment:
Sometimes the yield plugin's processing may take time and may not be successful. However, consecutive calls can be successful, so removing the throw.

Reply:
Can you add this explanation, or something even shorter as a comment here?

Reply:
Added